md-docs/user_guide/modules/rag_evaluation.md

```diff
@@ -1 +1,100 @@
-# RAG Evaluation
\ No newline at end of file
+# Rag Evaluation
+
+## Introduction
+
+RAG (Retrieval-Augmented Generation) is a way of building AI models that enhances their ability to generate accurate and contextually relevant responses by combining two main steps: **retrieval** and **generation**.
+
+1. **Retrieval**: The model first searches through a large set of documents or pieces of information to "retrieve" the most relevant ones based on the user query.
+2. **Generation**: It then uses these retrieved documents as context to generate a response, which is typically more accurate and aligned with the question than if it had generated text from scratch without specific guidance.
+
+Evaluating RAG involves assessing how well the model does in both retrieval and generation.
+
+The RAG evaluation module analyzes the three main components of a RAG framework:
+
+| Component  | Description                                                                        |
+| ---------- | ---------------------------------------------------------------------------------- |
+| User Input | The query or question posed by the user.                                           |
+| Context    | The retrieved documents or information that the model uses to generate a response. |
+| Response   | The generated answer or output provided by the model.                              |
+
+In particular, the analysis is performed on the relationships between these components:
```

Comment:
Da qua farei partire una nuova sezione che parli esplicitamente del nostro modulo.

Partirei quindi con il dire che è disponibile solo per i nostri Task (con link) di tipo RAG, e che è strutturata come report di evaluation di un set di dati.

Aggiungerei un blocco info che spieghi che è possibile farlo sia web app che da sdk.

Infine, ripartirei con le la frase di questo commento (in particular, the analysis....).

Qualcosa sul fatto che il report è visualizzabile in web app ma che si può anche esportare in formato excel.